<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Translator</class>
 <widget class="QMainWindow" name="Translator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>471</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>471</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Translator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>451</width>
      <height>331</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QComboBox" name="sourceComboBox">
      <property name="geometry">
       <rect>
        <x>98</x>
        <y>10</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="destComboBox">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>10</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>10</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&gt;</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="textToTranslate">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>421</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Fira Code</family>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="translatedText">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>421</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Fira Code</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="tabChangesFocus">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="translateButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>201</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Fira Code</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Tłumacz</string>
      </property>
     </widget>
     <widget class="QPushButton" name="correctButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>250</y>
        <width>201</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Fira Code</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Popraw tłumaczenie</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="msg">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>431</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QCheckBox" name="pl">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Polski</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="en">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Angielski</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="de">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Niemiecki</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="fr">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>131</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Francuski</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cs">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>131</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Czeski</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="es">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>131</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Hiszpański</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="ru">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>131</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Rosyjski</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="fi">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>131</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Fiński</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="ko">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>131</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Koreański</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="sv">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>131</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Szwedzki</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="ja">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Japoński</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="uk">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Ukraiński</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="ar">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Arabski</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="el">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Grecki</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
